### PR TITLE
Update Gitpod images at docs environment #9734

### LINKED ISSUE #9734: [BUG] change LinkFree images in GitPod guide to BioDrop
### Has this bug been raised before?

- [X] I have checked "open" AND "closed" issues and this is not a duplicate

### Where did you find this bug?

product on BioDrop

### Version of BioDrop (for example "v1.2.3")

2.84.4

### Description

For a new person, who wants to try the GitPod route for BioDrop, could find the 'GitPod' instructions confusing, because the following steps to create the BD profile refers to 'LinkFree' in the images and not BioDrop, in following the instructions: 
https://www.biodrop.io/docs/environments/gitpod

step 02: url implies this will be 'LinkFree'; not BD
step 03: New Workspace: refers to '/LinkFree'
step 04: VSCode says 'LinkFree' directory, not BG
step 02: Workspaces: references LinkFree in all links, not BG
See screen images below
I checked the 'Issues' by searching for 'gitpod'

### Screenshots

GitPod_BD_step01
GitPod_BD_step03
GitPod_BD_step04
GitPod_BD_step04_2wrksp
GitPod_BD01

### Do you want to work on this issue?

Yes

### If you want to work on this issue...

Proposal:
Update the images of the instructions to reflect 'BioDrop' instead of 'LinkFree'
This then negates any ambiguity.

### PR DESCRIPTION
**Pull request by dominic-source**
This commit will update images in the Gitpod doc environment.

## Fixes Issue

closes #9734

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by amandamartin-dev** [comment]
Hello! Can you please link the issue this PR will close.

**Comment by dominic-source** [comment]
#9734 

Here it is.

**Comment by SaraJaoude** [comment]
> #9734
> 
> Here it is.

Thank you @dominic-source  - in future could you please add this to the section in the Template description called `Fixes Issue`.  This allows for the issue to be closed automatically once the PR is merged, which helps maintainers manage the project.

**Comment by dominic-source** [comment]
Thanks so much, I apologise for all the delays. I'm new to open source, so I'll correct now on.
Thanks.

**Comment by dominic-source** [comment]
Hello, I wanted to ask if this pull request has been pulled. Because I notice it still appears here.

**Comment by ArvindParekh** [comment]
Hi @dominic-source. No it has not been merged yet, the maintainers will do that soon 👍🏻

File: data/dominic-source.json
Comment (kumarsonsoff3) [inline data/dominic-source.json:1]: You've created an empty file `data/dominic-source.json` which is not related to this issue. Could you please remove this file [here](https://github.com/dominic-source/BioDrop/delete/main/data/dominic-source.json).

And to create a profile using `json` you can follow [these docs](https://www.biodrop.io/docs/quickstart-json), & create a new PR from another branch.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by dominic-source** [comment]
Thanks for the observation
 I have deleted the file, please kindly confirm and provide me feedback.

Thanks & Regards

File: data/dominic-source.json
Comment (dominic-source) [reply data/dominic-source.json:1]: Thanks for the observation. 
I have deleted the file, please kindly confirm and provide me feedback.

Thanks and Regards.

**Pull request merged**
